terraform/dev/main.tf

```diff
@@ -50,8 +50,8 @@ data "cloudfoundry_service" "rds" {
 resource "cloudfoundry_service_instance" "database" {
   name             = "tdp-db-dev"
   space            = data.cloudfoundry_space.space.id
-  service_plan     = data.cloudfoundry_service.rds.service_plans["micro-psql"]
-  json_params      = "{\"version\": \"15\"}"
+  service_plan     = data.cloudfoundry_service.rds.service_plans["medium-gp-psql"]
+  json_params      = "{\"version\": \"15\", \"storage_type\": \"gp3\", \"storage\": 50}"
```

Comment:
@elipe17 will we need to update [staging](https://github.com/raft-tech/TANF-app/blob/93bd544b6bb52ef66d0b7afd6aa86debcdd76c31/terraform/staging/main.tf#L53L54) and [prod's storage type](https://github.com/raft-tech/TANF-app/blob/93bd544b6bb52ef66d0b7afd6aa86debcdd76c31/terraform/production/main.tf#L54)?

Reply:
If we plan to run big reparse events in the staging environment we should consider upgrading it. I do think we should move prod up to a `large` or `xlarge` with the same storage type. I believe the reparse swaths in prod will be larger than what you tested in staging right?

Reply:
Another issue I noticed during the reparse is that elastic went offline several times. Im assuming we were hitting the AWS rate limits for that service type just like we were for postgres. Hitting those rate limits doesnt break anything but it makes the reparse very very slow because elastic allows five one minute timeouts when the connection is lost. That is, we could add five extra minutes per `bulk_create_records` when elastic goes down.